poker/utils/player.py

```diff
@@ -0,0 +1,74 @@
+import random
+
+from poker.utils.chip import PlayerStack
+from poker.utils.deck import Deck, Card
+from poker.utils.action import Action
+
+
+class Player:
+
+    PLAYER = "Player"
```

Comment:
enum?

Reply:
like you did with Action.Constant below?

Reply:
You got it. I'll work on this update. Thanks.